modules/auxiliary/scanner/http/trace.rb

```diff
@@ -27,36 +31,36 @@ def run_host(target_host)
     begin
       res = send_request_raw({
         'version'      => '1.0',
-        'uri'          => '/',
+        'uri'          => '/<script>alert(1337)</script>', #XST Payload
         'method'       => 'TRACE',
         'headers' =>
         {
-          'Cookie' => "did you echo me back?",
+          'Cookie' => "TRACE ME PLS",
```

Comment:
If you don't need the cookie (you mentioned on the previous PR you don't), please remove it.